cves/kernel/CVE-2015-8787.yml

```
@@ -320,8 +339,10 @@ vouch:
 
     Answer must be true or false.
     Write a note about how you came to the conclusions you did, regardless of what your answer was.
-  answer:
-  note:
+  answer: true
+  note: |
+    Both the original commit as well as the fix commit had several developers
+    sign off on them'
```

The wording could be improved, but at least add a period.

```suggestion
    sign off on them.
```